frontstage/views/surveys/upload_survey.py

```diff
@@ -62,10 +62,23 @@ def upload_survey(session):
 
     try:
         # Upload the file to the collection instrument service
-        collection_instrument_controller.upload_collection_instrument(
+        error_text = collection_instrument_controller.upload_collection_instrument(
             upload_file, case, business_party, party_id, survey
         )
+        if error_text is not None:
```

Comment:
Shouldn't this be "if error_text"?  You wouldn't need the 'is not None' then

Reply:
There's fundamentally no difference so it's a matter of taste